tools/azure-sdk-tools/ci_tools/ci_interactions.py

```diff
@@ -0,0 +1,37 @@
+import os, inspect
+from ci_tools.variables import in_ci
+
+
+def output_ci_warning(message: str, location=None) -> None:
+    ci_type = in_ci()
```

Comment:
I understand why this is being done, but it does seem a little weird. I would consider making the `TF_BUILD`/`CI` check come from a different method

Reply:
Yeah fair enough. There is no real logic to share here, but I'll look to do _something_ about that in a further PR. I kinda want to make `in_ci` return an enum value, but I haven't done the research to make that happen elegantly.